### PR TITLE
Fixes mice being able to interact with fire alarms

### PR DESCRIPTION
**Pull request by ShiftyRail**
Requested by @TGFH

### CODE REVIEW

**Comment by TGFH** [comment]
<3 u shifty.

File: code/game/machinery/alarm.dm [hunk 1]
```diff
@@ -1214,6 +1214,8 @@ FIRE ALARM
 	return ..()
 
 /obj/machinery/firealarm/CtrlClick(var/mob/user)
+	if (ismouse(user)) // Squeak
```
Comment (Kurfursten) [inline code/game/machinery/alarm.dm:1217]: a dex check might make more sense, or ishigherbeing

Comment (ShiftyRail) [reply code/game/machinery/alarm.dm:1217]: Wouldn't that make monkeys unable to use it ? Given that `attack_paw` calls `attack_hand`, it seems they were intended to use it.

Comment (Kurfursten) [reply code/game/machinery/alarm.dm:1217]: Higherbeing allows monkeys with methylin

You could also do higher being or monkey. To eliminate use by cows.

Comment (ShiftyRail) [reply code/game/machinery/alarm.dm:1217]: Alright

**Comment by BurntDevil** [comment]
Yes, this is annoying, but at this rate, why do we have mice if we are actively taking away everything they can do, while making them generally unplayable with stuff like the food rate consumption, the mommi-vent nerf that carried to mice, and other obnoxious changes over the last 3 years? The sole improvement to mice has been spiderbot riding, and even that is broken because you cant ventcrawl in a spiderbot despite spiderbots and mice both normally being able to.

You ought be replacing things you take from them with new things they can actually do, otherwise theres very little reason to even have them.

**Comment by jknpj** [comment]
>why do we have mice if we are actively taking away everything they can do

Because they weren't supposed to do this, alarms just have no sanity. Also, reminder:

https://github.com/vgstation-coders/vgstation13/blob/e9a806f30b4db0efa2a68b9eb42e3120d2321b6a/code/modules/mob/dead/observer/verbs.dm#L166

**Comment by ShiftyRail** [comment]
Done.

**Pull request merged**